### PR TITLE
Autogenerate more files

### PR DESCRIPTION
**Pull request by pakrym-stripe**
Switch line item-related files to be auto-generated.

r? @dcr-stripe

### CODE REVIEW

File: src/Stripe.net/Services/InvoiceItems/InvoiceItemPriceDataOptions.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        public string Product { get; set; }
+
+        /// <summary>
+        /// Specifies whether the price is considered inclusive of taxes or exclusive of taxes. One
+        /// of <c>inclusive</c>, <c>exclusive</c>, or <c>unspecified</c>. Once specified as either
+        /// <c>inclusive</c> or <c>exclusive</c>, it cannot be changed.
+        /// One of: <c>exclusive</c>, <c>inclusive</c>, or <c>unspecified</c>.
+        /// </summary>
+        [JsonProperty("tax_behavior")]
+        public string TaxBehavior { get; set; }
```
Comment (dcr-stripe) [inline src/Stripe.net/Services/InvoiceItems/InvoiceItemPriceDataOptions.cs:29]: Double checking: this was excluded due to being manually maintained and no other reason, right?

File: src/Stripe.net/Services/Plans/PlanProductOptions.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        /// statement in capital letters. Non-ASCII characters are automatically stripped.
+        /// </summary>
+        [JsonProperty("statement_descriptor")]
+        public string StatementDescriptor { get; set; }
+
+        /// <summary>
+        /// A <a href="https://stripe.com/docs/tax/tax-categories">tax code</a> ID.
+        /// </summary>
+        [JsonProperty("tax_code")]
+        public string TaxCode { get; set; }
```
Comment (dcr-stripe) [inline src/Stripe.net/Services/Plans/PlanProductOptions.cs:53]: Same check here.

File: src/Stripe.net/Services/Subscriptions/SubscriptionItemPriceDataOptions.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        public SubscriptionItemPriceDataRecurringOptions Recurring { get; set; }
+
+        /// <summary>
+        /// Specifies whether the price is considered inclusive of taxes or exclusive of taxes. One
+        /// of <c>inclusive</c>, <c>exclusive</c>, or <c>unspecified</c>. Once specified as either
+        /// <c>inclusive</c> or <c>exclusive</c>, it cannot be changed.
+        /// One of: <c>exclusive</c>, <c>inclusive</c>, or <c>unspecified</c>.
+        /// </summary>
+        [JsonProperty("tax_behavior")]
+        public string TaxBehavior { get; set; }
```
Comment (dcr-stripe) [inline src/Stripe.net/Services/Subscriptions/SubscriptionItemPriceDataOptions.cs:35]: Same check here.

**Review by dcr-stripe — APPROVED** [review]
LGTM, thank you for doing this @pakrym-stripe ! Just a few things to double check and then good to merge.

I'm also assuming here that the file move itself isn't breaking since it's still all under the Stripe namespace.

File: src/Stripe.net/Services/InvoiceItems/InvoiceItemPriceDataOptions.cs (hunk 1 above)
Comment (pakrym-stripe) [reply src/Stripe.net/Services/InvoiceItems/InvoiceItemPriceDataOptions.cs:29]: I suspect so. We didn't suppress this property in any other generated file.

File: src/Stripe.net/Services/Plans/PlanProductOptions.cs (hunk 2 above)
Comment (pakrym-stripe) [reply src/Stripe.net/Services/Plans/PlanProductOptions.cs:53]: Yep, we have `tax_code` in many other places.

File: src/Stripe.net/Services/Subscriptions/SubscriptionItemPriceDataOptions.cs (hunk 3 above)
Comment (pakrym-stripe) [reply src/Stripe.net/Services/Subscriptions/SubscriptionItemPriceDataOptions.cs:35]: Same as in `InvoiceItemPriceDataOptions.cs`

**Pull request merged**